.gitlab/source_test/ebpf.yml

```diff
@@ -35,6 +35,7 @@ tests_ebpf:
     # Compile runtime security stress tests to be executed in kitchen tests
     - inv -e security-agent.build-stress-tests --output=$DD_AGENT_TESTING_DIR/site-cookbooks/dd-security-agent-check/files/stresssuite
     # Compile master version for comparison, uncomment following lines when merged
+    - git diff
```

Comment:
Is this required or is it there just for debugging?

Reply:
Indeed was just for debugging. Will removed it